github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -481,7 +479,7 @@ periodics:
           - name: TARGET
             value: "k8s-1.20"
           - name: KUBEVIRT_E2E_SKIP
-            value: "SRIOV|GPU|Operator|\\[sig-network\\]|\\[sig-storage\\]|owner:@sig-compute"
+            value: "SRIOV|GPU|Operator|\\[sig-network\\]|\\[sig-storage\\]|sig-compute"
```

Comment:
Better if we can please keep one style `\\[sig-compute\\]`

Reply:
Changed